src/html_parser.cpp

```
@@ -12,668 +12,766 @@
+		else if (tag_id == LXB_TAG_SPAN)
+		{
+			owner().sendTag(tag::Span{ .styling = html_node_styling(node) });
+		}
+		else if (tag_id == LXB_TAG_A)
+		{
+			const lxb_char_t* href_attr = lxb_dom_element_get_attribute(const_cast<lxb_dom_element_t*>(element), (const lxb_char_t*)"href", 4, nullptr);
+			std::string url;
+			if (href_attr) {
+				url = std::string((const char*)href_attr);
+				//skip target if begins with "#" or "javascript"
+				if (url[0] == '#' || url.find("javascript") == 0)
```

Cppcheck (2.10-2)

[performance] Inefficient usage of string

(stlIfStrFind)